src/pages/Anix/main.ts

```diff
@@ -0,0 +1,127 @@
+import { Animeworld } from '../Animeworld/main';
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Anix: pageInterface = {
+  domain: 'https://anix.to',
+  languages: ['English'],
+  name: 'Anix',
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return true;
+  },
+  sync: {
+    getTitle(url: string): string {
+      return j.$('h1.anime-name').text().trim();
+    },
+    getIdentifier(url: string): string {
+      const anime = url.split('/')[4];
+      const animeWords = anime.split('-');
+      return animeWords[animeWords.length - 1];
+    },
+    getOverviewUrl(url: string): string {
+      return utils.absoluteLink(
+        j.$('div.range-wrap > div.range > div > a').first().attr('href'),
+        Anix.domain,
+      );
+    },
+    getEpisode(url: string): number {
+      return parseInt(j.$('div.range-wrap > div.range > div > a.active').text());
+    },
+    nextEpUrl(url: string): string | undefined {
+      return j
+        .$('div.range-wrap > div.range > div > a.active')
```

Comment:
Too specific. As it is the same as above it could be defined as a const. This way there is no repeated code

Reply:
Added const `activeA`.